src/Backend/DependencyInjection/BackendExtension.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Backend\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+
+class BackendExtension extends Extension implements PrependExtensionInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        // Nothing needs to be loaded here
+    }
```

Comment:
it is, I tried to remove it first, but that gave an error because the method needs to be implemented